sklearn/metrics/tests/test_score_objects.py

```diff
@@ -8,19 +8,25 @@
 
 from sklearn.utils.testing import assert_almost_equal
 from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
 from sklearn.utils.testing import assert_raises
 from sklearn.utils.testing import assert_raises_regexp
 from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import assert_false
 from sklearn.utils.testing import ignore_warnings
 from sklearn.utils.testing import assert_not_equal
 from sklearn.utils.testing import assert_warns_message
 
 from sklearn.base import BaseEstimator
 from sklearn.metrics import (f1_score, r2_score, roc_auc_score, fbeta_score,
                              log_loss, precision_score, recall_score)
-from sklearn.metrics import cluster as cluster_module
-from sklearn.metrics.scorer import (check_scoring, _PredictScorer,
-                                    _passthrough_scorer)
+from sklearn.metrics import accuracy_score
+from sklearn.metrics import cluster
+from sklearn.metrics.scorer import check_scoring
+from sklearn.metrics.scorer import (
+    _check_multimetric_scoring as check_multimetric_scoring)
```

Comment:
Install a code editor with such refactoring tools built in if you wish, but don't make the code look strange for such reasons.